### DISCUSSION TITLE
iCUE Lighting Node PRO RGB Lighting Controller

### DISCUSSION THREAD

**Discussion by moonman1425** [EvanMulawski/FanControl.CorsairLink#16] [Ideas]
I was wondering if you would consider adding support for the iCUE Lighting Node PRO RGB Lighting Controller?

**Comment by EvanMulawski** [comment]
Hi @moonman1425, the Lighting Node PRO only handles lighting. Fan Control only controls fan speed. RGB fans have one cable for the fan speed control (usually 4-pin PWM) and a separate cable for RGB lighting. You would connect your fan's RGB cable to the Lighting Node PRO and its PWM cable to a fan controller or a fan header on your motherboard.
